src/seedu/addressbook/data/person/Name.java

```diff
@@ -60,4 +60,9 @@ public int hashCode() {
         return fullName.hashCode();
     }
 
+	@Override
+	public String getPrintableString() {
```

Comment:
Correct implementation of the interface. 

As discussed in tutorial, you may want to consider the privacy setting when returning a printable string in this system.